packages/pwa-buildpack/lib/Utilities/fetcherUtils.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+
+const fetchQuery = query =>
```

Comment:
I suppose this is okay for now, we may want to use Apollo client with `buildpack` eventually since it bakes in a bunch of best practices. We won't really need it for the cache though, so it may be overkill.

Reply:
We could if we have more GQL calls to make at build time which is what buildpack does. If not, we will be having a dependency just to make a call.

Reply:
It's not an additional dependency. We already know that apollo-client will be available in the project because it's a peer dependency. However, it's an optimization to replace this with apollo-client and I'm happy to merge this as-is.